spec/support/helpers/api_helper.rb

```diff
@@ -147,4 +148,14 @@ def stub_keyworker(prison_code, offender_id, keyworker)
   def reload_page
     visit current_path
   end
+
+  def stub_community_offender(nomis_offender_id, community_data)
+    stub_request(:get, "#{COMMUNITY_HOST}/offenders/nomsNumber/#{nomis_offender_id}/all").
+        to_return(body: community_data.to_json)
+  end
+
+  def stub_community_registrations(nomis_offender_id, registrations)
+    stub_request(:get, "#{COMMUNITY_HOST}/offenders/nomsNumber/#{nomis_offender_id}/registrations").
+        to_return(body: registrations.to_json)
```

Comment:
The response `body` should be a Hash with a nested `registrations` Array.

### For example, given an offender with 'no registrations':

The response `body` from this stub is:

```json
[]
```

But the actual response from the Community API is:

```json
{
  "registrations": []
}
```